proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,121 @@
+2. The articles aim to:
+   1. Distribute Mixxx as a free and open source software
+   2. Distribute Manuals and Guides
+   3. Organize user support and education 
+   4. Mentor students during interneships for learning software developmen.
+   5. Maintaining the development infrstructur for easy code access
+3. Amendments to the association’s purpose may only be made in accordance with § 3 (1).
+
+## 3. Non-profit Association
+
+1. The Association’s activities are exclusively non-profit and tax-privileged in accordance with German Tax Regulations. Its activities are for the common good and not for profit-making.
+2. Financial means may only be used for purposes stipulated in the Articles of Association. Members are reimbursed for expenses but do not receive direct benefits from the Association’s funds.
+3. It is not permitted to favor someone through association expenditures that do not serve the association's purpose or are disproportionately high.
+
+## 4. Types of Membership and Members of the Association
+
+1. Membership is open to natural persons and legal bodies supporting the association's aims. Membership types include:
+   1. Active members: Natural persons actively supporting the Association’s purpose and aims.
+   2. Extraordinary members: Natural persons and legal bodies supporting the Association’s aims but without voting rights.
+   3. Supporting Members: Extraordinary members primarily supporting through financial or material contributions.
+2. Active membership is granted upon suggestion and support by active members or a majority vote. Criteria include long-term commitment and contributions.
+3. Membership terminates due to exit, expulsion, death, or loss of legal status. Written requests for termination must be submitted at least 4 weeks prior to December 31st.
+4. Expulsion may occur due to serious violations or non-payment of fees over 12 months. Members must be given the opportunity to justify their actions before expulsion.
+5. Members leaving the Association or if the Association is liquidated do not have a right to reimbursement for contributed assets.
+
+## 5. Organs of the Association
+
+- The General Assembly
+- The Board
+
+## 6. General Assembly
+
+1. The General Assembly consists of all active members, each with one vote.
+2. It takes place at least once a year, with invitations sent at least six weeks prior.
+3. An extraordinary General Assembly may be called if requested by at least 20% of members.
+4. The General Assembly has a quorum if the invitation was submitted in proper form.
+5. A ¾ majority of votes is required to amend the Articles of Association or decide the liquidation of the Association.
+6. Active members unable to attend may be represented by another member with written authorization.
```

Comment:
```suggestion
6. Active members unable to attend may be represented by another active member with written authorization.
```

Reply:
whats the criteria for a member to be considered active?

Reply:
It's defined in 4.1.1, but I suggest that we call it "ordinary member" instead to avoid this kind of ambiguity.

Reply:
@JoergAtGithub I don't understand your suggestion. If you're an active member and cannot participate, why should you name another active member to participate on your behalf? That member may already participate on its own, so it would be a bit pointless. Or would this member then have 2 votes?

Is there a specific reason why an active member is not allowed to name an extraordinary/supporting member as stand-in?

Reply:
Yes, than this active member would have two votes.

I want to ensure, that we can not loose control over the e.V. - a project like Mixxx will not work, if the end users vote what the developers have to do.